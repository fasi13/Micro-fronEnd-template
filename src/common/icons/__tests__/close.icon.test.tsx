import { render } from '@testing-library/react';
import React from 'react';
import { CloseIcon } from '../index';

test('It should render CloseIcon', async () => {
	const size = { width: 512, height: 512 };

	expect(
		render(<CloseIcon className="" width={size.width} height={size.height} />),
	).toMatchSnapshot();
});
