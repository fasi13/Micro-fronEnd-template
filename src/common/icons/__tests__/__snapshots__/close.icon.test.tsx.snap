// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render CloseIcon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        class=""
        fill="none"
        height="512"
        viewBox="0 0 329 329"
        width="512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          clip-path="url(#clip0)"
        >
          <path
            d="M194.641 164.769l128.107-128.11c8.336-8.333 8.336-21.806 0-30.14-8.333-8.332-21.807-8.332-30.14 0L164.498 134.63 36.392 6.52c-8.337-8.333-21.806-8.333-30.14 0-8.336 8.333-8.336 21.806 0 30.14L134.36 164.768 6.253 292.879c-8.337 8.333-8.337 21.807 0 30.14a21.252 21.252 0 0015.07 6.245 21.259 21.259 0 0015.07-6.245l128.105-128.11 128.11 128.11a21.26 21.26 0 0015.07 6.245 21.26 21.26 0 0015.07-6.245c8.336-8.333 8.336-21.807 0-30.14l-128.107-128.11z"
            fill="#fff"
          />
        </g>
        <defs>
          <clippath
            id="clip0"
          >
            <path
              d="M0 0h329v329H0z"
              fill="#fff"
            />
          </clippath>
        </defs>
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      class=""
      fill="none"
      height="512"
      viewBox="0 0 329 329"
      width="512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        clip-path="url(#clip0)"
      >
        <path
          d="M194.641 164.769l128.107-128.11c8.336-8.333 8.336-21.806 0-30.14-8.333-8.332-21.807-8.332-30.14 0L164.498 134.63 36.392 6.52c-8.337-8.333-21.806-8.333-30.14 0-8.336 8.333-8.336 21.806 0 30.14L134.36 164.768 6.253 292.879c-8.337 8.333-8.337 21.807 0 30.14a21.252 21.252 0 0015.07 6.245 21.259 21.259 0 0015.07-6.245l128.105-128.11 128.11 128.11a21.26 21.26 0 0015.07 6.245 21.26 21.26 0 0015.07-6.245c8.336-8.333 8.336-21.807 0-30.14l-128.107-128.11z"
          fill="#fff"
        />
      </g>
      <defs>
        <clippath
          id="clip0"
        >
          <path
            d="M0 0h329v329H0z"
            fill="#fff"
          />
        </clippath>
      </defs>
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
